app/views/physical_server/show.html.haml

```diff
@@ -1,5 +1,12 @@
+- @angular_form = true
```

Comment:
Cool so please, remove it we should not be adding lines that are not needed. It makes future refactoring work more difficult.

Thanks for your explanations.

I am happy that at least some of my comment make sense with my limited Angular skills ;-)